### PR TITLE
Enable webtests

### PR DESCRIPTION
**Pull request by markbandstra**
I would like to enable the webtests for Travis CI so that we can track problems accessing web APIs like we had with the XCOM tools. In fact, I have already enabled weekly builds for this purpose. This PR is to make sure the webtests work in Travis and to fix any outstanding webtest issues.

### CODE REVIEW

**Comment by markbandstra** [comment]
This is probably all I want to do with this PR, but I am open to input.

**Comment by markbandstra** [comment]
Could someone take a look at this? Thanks.

File: tests/isotope_test.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def test_isotope_str(iso_str, sym, A, m):
 
 
 ISOTOPE_PROPERTIES = [
-    ('H-3', (3.888E8, False, None, '1/2+', 0., None, [['B-'], [100.]],)),
+    ('H-3', (3.888E8, False, None, '1/2+', 0., 14.94980957, [['B-'], [100.]],)),
```
Comment (jccurtis) [inline tests/isotope_test.py]: Why was this `None` before? I'm unfamiliar with this test...

Comment (markbandstra) [inline tests/isotope_test.py]: That's the mass excess, and apparently the database used to have an empty entry for it.

**Comment by jccurtis** [comment]
Besides my previous comment this looks good to me.

**Pull request merged**